x-pack/plugins/uptime/server/lib/requests/get_journey_screenshot.test.ts

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getJourneyScreenshot } from './get_journey_screenshot';
+import { mockSearchResult } from './helper';
+
+describe('getJourneyScreenshot', () => {
+  it('returns screenshot data', async () => {
+    const screenshotResult = {
+      _id: 'id',
+      _source: {
+        synthetics: {
+          blob_mime: 'image/jpeg',
+          blob: 'image data',
+          step: {
+            name: 'load homepage',
+          },
+          type: 'step/screenshot',
+        },
+      },
+    };
+    expect(
+      await getJourneyScreenshot({
+        uptimeEsClient: mockSearchResult([], {
+          // @ts-expect-error incomplete search result
+          step: { image: { hits: { hits: [screenshotResult] } } },
+        }),
+        checkGroup: 'checkGroup',
+        stepIndex: 0,
+      })
+    ).toMatchInlineSnapshot(`
```

Comment:
I think this output is straightforward enough that to write as a `toEqual` versus a snapshot

Reply:
See https://github.com/elastic/kibana/pull/101678/commits/b7ff654433fea4ff409b33e44758d1d32ad50975.